### DISCUSSION TITLE
Tapo S200D

### DISCUSSION THREAD

**Discussion by Mwalton4uk** [petretiandrea/home-assistant-tapo-p100#559] [Q&A]
Hello, can someone tell me if the integration supports the S200D?

If I need to buy hub for it to work that's fine but before I buy I would like to know.

Want to put 2 next to each other having one control a light and the other controlling fan speed via HA

**Comment by ashwinikalantri** [comment]
I have the same question. I feel the hub will be needed. But I want to understand if the switch will be reported in HA and will it control other devices? I want it to control my fan (sonoff ifan4).

**Comment by RobinCu** [comment]
Once the H100 hub is added to HA, the S200Ds appear as an entity (though they are shown as S200B).  The only values reported are: 
- <unknown entity> battery low, 
- <unknown entity> battery normal 
- report interval duration changes. 

Therefore, it's not possible to use the button to trigger events.
